### DISCUSSION TITLE
Does the payload support space?

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#869] [Q&A] [answered]
just like: [https://www.contentful.com/help/spaces-and-organizations/](https://www.contentful.com/help/spaces-and-organizations/)

**Comment by DanRibbens** [comment, marked as answer]
Interesting! Can you explain your use-case for something like this?

Payload has open ended user models and access control for you to define for your own needs. As you're able to do this on a per-project basis I'm not sure how it fits into the core product. It seems like a candidate for some kind of plugin, at least that is my initial reaction without any context.
